pkg/actors/actors.go

```diff
@@ -1199,7 +1202,7 @@ func (a *actorsRuntime) executeTimer(actorType, actorID, name, dueTime, period,
 	req := invokev1.NewInvokeMethodRequest(fmt.Sprintf("timer/%s", name))
 	req.WithActor(actorType, actorID)
 	req.WithRawData(b, invokev1.JSONContentType)
-	_, err = a.callLocalActor(context.Background(), req)
+	_, err = a.Call(context.Background(), req)
```

Comment:
Please revert this back to `callLocalActor` as per https://github.com/dapr/dapr/pull/4072#discussion_r777060857.

Otherwise, this looks good to me.